spring-boot/proxy/src/main/java/eu/coatrack/proxy/filters/post/ErrorLoggingFilter.java

```diff
@@ -80,7 +84,7 @@ public Object run() {
         log.debug(String.format("Erroneous response status is: %s", servletResponse.getStatus()));
 
         if (ctx.getResponseBody() == null && ctx.getResponseDataStream() == null) {
-            log.warn("Response body and data stream are null - assuming that service is unavailable (logging status 503)");
+            log.error(String.format("An unexpected error occurred when your CoatRack gateway contacted the central CoatRack portal in order to verify an API key that was sent by a client: Service is not unavailable/accessible, please check if the service is accessible and gateway config in %s/admin/proxies", adminServerUrl));
```

Comment:
Please check wording: "not unavailable" is confusing. Besides, we never know if it is a network issue or if CoatRack admin is actually down. The message should only say what we know: that the gateway was not able to properly communicate with admin.